bluemira/codes/_freecadapi.py

```diff
@@ -232,13 +232,53 @@ def make_bezier(points: Union[list, np.ndarray], closed: bool = False) -> Part.W
 
 
 def make_bspline(
+    poles, mults, knots, periodic, degree, weights, check_rational
+) -> Part.Wire:
+    """
+    Builds a B-Spline by a lists of Poles, Mults, Knots
+
+    Parameters
+    ----------
+    poles: Union[list, np.ndarray]
+        list of poles.
+    multis: Union[list, np.ndarray]
+        list of integers for the multiplicity
+    knots: Union[list, np.ndarray]
+        list of knots
+    periodic: Bool
+    degree: int
+        bspline degree
+    weights: Union[list, np.ndarray]
+        sequence of float
+    check_rational: Optional[Iterable]
```

Comment:
Should this have type `bool`? The `isRational` call below would suggest a boolean to me